scripts/systemd/gogs.service

```diff
@@ -8,19 +9,43 @@ After=network.target
 #After=redis.service
 
 [Service]
-# Modify these two values and uncomment them if you have
-# repos with lots of files and get an HTTP error 500 because
-# of that
-###
-#LimitMEMLOCK=infinity
-#LimitNOFILE=65535
-Type=simple
+Restart=on-failure
+
+; User and group the process will run as.
 User=git
 Group=git
+ 
+; Gogs directoryies.
+Environment=USER=git HOME=/home/git
 WorkingDirectory=/home/git/gogs
+ 
 ExecStart=/home/git/gogs/gogs web
-Restart=always
-Environment=USER=git HOME=/home/git
+
+; Repos with lots of files and get an HTTP error 500 will need that
+;LimitMEMLOCK=infinity
+;LimitNOFILE=65535
+ 
+; Use private /tmp and /var/tmp, which are discarded after gogs stops.
+PrivateTmp=true
+ 
+; Use a minimal /dev
+PrivateDevices=true
+ 
+; Hide /home, /root, and /run/user. Nobody will steal your SSH-keys.
+ProtectHome=true
```

Comment:
This will make Gogs crash since all repos by default are in `/home/git/gogs-repositories`.